src/CloudFoundry.VisualStudio/Controls/RoutesUserControl.xaml.cs

```diff
@@ -30,16 +31,26 @@ private void btnAddHost_Click(object sender, RoutedEventArgs e)
         {
             if (string.IsNullOrWhiteSpace(tbName.Text) == false)
```

Comment:
Validations should happen in https://github.com/cloudfoundry-incubator/cf-vs-extension/blob/develop/src/CloudFoundry.VisualStudio/ProjectPush/PublishProfileEditorResources.cs